build_for_android.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/sh
+
+if [ "x$1" = "x--help" ] || [ "x$1" = "xhelp" ] || [ "x$1" = "x-h" ]; then
+  echo "## Before building:"
+  echo "    # run autogen.sh if \"./configure\" does not exist"
+  echo "    ./autogen.sh"
+  echo
+  echo "## To build without an NDK standalone toolchain, and without libreadline:"
+  echo "    # For given NDK directory <NDKPath>, execute from the scanmem source directory:"
+  echo "    ./configure --without-readline"
+  echo "    <NDKPath>/ndk-build>"
+  echo "or"
+  echo "    export NDK_HOME=\"<NDKPath>\""
+  echo "    ./build_for_android.sh"
+  echo
+  echo "## To build with standalone toolchain:"
+  echo "    export NDK_STANDALONE_TOOLCHAIN=\"/your/toolchain/path\""
+  echo "    export HOST=\"your-androideabi\" # Default arm-linux-androideabi"
+  echo "    ./build_for_android.sh"
+  echo
+  echo "## Advanced features and Environment variables that may be set..."
+  echo "NDK_STANDALONE_TOOLCHAIN - A standalone toolchain is required to build full capabilities."
+  echo "NDK_HOME                 - This will allow to use the ndk-build, and a full standalone toolchain is not required.  If this is set, the scanmem executable will be built with ndk-build, and not the standalone toolchain.  This needs to be used if the target Android version is below 5.0.  This requirement may be fixed in the future."
```

Comment:
Ok.  Is it also good to have help coupled with the script, or is a .md file better?